test/IECoreScene/SceneInterfaceTest.py

```diff
@@ -87,39 +87,47 @@ def testErase( self ) :
 
 		self.writeSCC()
 
-		instance1 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
-		instance2 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
+		instance1 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
+		instance2 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
 		self.assertTrue( instance1.isSame( instance2 ) )
 
-		IECoreScene.SharedSceneInterfaces.erase( SceneInterfaceTest.__testFile )
+		IECoreScene.SharedSceneInterfaces.erase( self.__testFile )
 
-		instance3 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
+		instance3 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
 		self.assertFalse( instance3.isSame( instance1 ) )
 
-		instance4 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
+		instance4 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
 		self.assertFalse( instance4.isSame( instance1 ) )
 		self.assertTrue( instance4.isSame( instance3 ) )
 
 	def testClear( self ) :
 
 		self.writeSCC()
 
-		instance1 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
-		instance2 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
+		instance1 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
+		instance2 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
 		self.assertTrue( instance1.isSame( instance2 ) )
 
 		IECoreScene.SharedSceneInterfaces.clear()
 
-		instance3 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
+		instance3 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
 		self.assertFalse( instance3.isSame( instance1 ) )
 
-		instance4 = IECoreScene.SharedSceneInterfaces.get( SceneInterfaceTest.__testFile )
+		instance4 = IECoreScene.SharedSceneInterfaces.get( self.__testFile )
 		self.assertFalse( instance4.isSame( instance1 ) )
 		self.assertTrue( instance4.isSame( instance3 ) )
 
 	def testVisibilityName( self ) :
 		self.assertEqual( IECoreScene.SceneInterface.visibilityName, "scene:visible" )
 
+	def setUp( self ) :
+		self.tempDir = tempfile.mkdtemp()
+		__testFile = os.path.join( self.tempDir, "test.scc" )
+		__testFileUpper = os.path.join( self.tempDir, "test.SCC" )
```

Comment:
I'm surprised this is working, shouldn't these need to be `self.__testFile` and `self.__testFileUpper`?

Reply:
Right, that needs the `self.` added. It looks like `SceneInterfaceTest` is not in `All.py`, I will add it.